### DISCUSSION TITLE
Romanian translation

### DISCUSSION THREAD

**Discussion by Alexandru51** [d4rken-org/sdmaid-se#531] [Q&A] [answered]
Hello, i want the romanian language, like how SD Maid 1 has.

**Comment by d4rken** [comment, marked as answer]
Thanks!

Translation files are not yet available, but you can already join the translation project to get notified when it's possible :)

https://crowdin.com/project/sdmaid-se

**Comment by Alexandru51** [comment]
Understandable, thanks for saying :)
